packages/lib/src/components/FeedbackWidget/FeedbackWidget.tsx

```diff
@@ -242,7 +243,7 @@ export function FeedbackWidget({
               return (
                 <Stack key={key}>
                   <ActionIcon
-                    color="blue"
+                    color={theme.primaryColor}
```

Comment:
We don't even need to specify it explicitly since it's set to the primary color by default. For `ActionIcon` it was necessary as discussed for Mantine v6, v7 does fix it. 🙂

Reply:
Sure. I thought we had decided to use it until the update to Mantine 7 was complete. But since it should be finished soon, this should be fine. I removed the color prop, so the ActionIcons are grey for now.

Reply:
I thought we decided to already remove it haha, no worries morries.